docs/conf.py

```diff
@@ -50,16 +50,16 @@
 
 # General information about the project.
 project = u'persistent'
-copyright = u'2011, ZODB Developers <zope-dev@zope.org>'
+copyright = u'2011,2016 ZODB Developers <zope-dev@zope.org>'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '3.10'
+version = '4.2'
 # The full version, including alpha/beta/rc tags.
-release = '3.10b1'
+release = '4.2.2'
```

Comment:
This is not DRY. It was forgotten to update for years. The only way to make it DRY seems to be using this `version.txt` approach.